### ISSUE TITLE
Epochs question

### ISSUE THREAD

**Issue by l-cr** [facebookresearch/DiT#56]
training with imagenet, and batchsize is 256, so how many the epochs?

**Comment by devilkadabra69** [comment]
it mainly depends on the following factors 

1. model architecture(type of data basically refers to the structure)
2. learning rate(θnew​=θold​−learning rate×gradient   -> it's an hyper parameter)
3. data augmentation


so try to start with 45 and eventually increase it to the point before the model starts overfitting 

45 -> 50 this range should be great to start with 

extra note :- you may check with 35->40 also to compare the result which got from 45->50 if first one is better ,consider decreasing from 45 -> 44 ->43 .... 

hope it helps !

**Comment by zhtmike** [comment]
7000k steps mentioned in paper, 7000000 * 256 / 1281167 ~ 1400 epoches, mentioned in the code

**Comment by l-cr** [comment]
> 7000k steps mentioned in paper, 7000000 * 256 / 1281167 ~ 1400 epoches, mentioned in the code

@zhtmike thank u
